bamazonCustomer.js

```diff
@@ -0,0 +1,91 @@
+var mysql = require("mysql");
+var inquirer = require("inquirer");
+
+var data;
+
+var connection = mysql.createConnection({
+  host: "localhost",
+  port: 3306,
+  user: "root",
+  password: "eqk8S8GB6fj8",
+  database: "bamazon_DB"
+});
+
+connection.connect(function(err) {
+  if (err) throw err;
+  runSearch();
+});
+
+function runSearch() {
+  var query = "SELECT item_id,product_name,department_name,price,stock_quantity FROM products;";
+  connection.query(query, function(err, res) {
+    data = res;
+    for (var i = 0; i < res.length; i++) {
+      console.log(
+        "Product ID: " +
+        res[i].item_id +
+        " || Product Name: " +
+        res[i].product_name +
+        " || Department: " +
+        res[i].department_name +
+        " || Price: " +
+        res[i].price +
+        " || Stock Quantity: " +
+        res[i].stock_quantity
+      );
+    }
+    promptUser();
```

Comment:
Also, instead of making data a global variable as they are bad practice now, I would just write it like this:
```javascript
var data = res
promptUser(data)```
this way you don't have data sitting in the global scope and it is more concise as to where the data is supposed to go.